alert-database/src/main/resources/liquibase/6.9.0/migrate-proxy-config.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+    <!-- create an initial row -->
+    <changeSet author="martinch" id="create-initial-proxy-config-entries">
+        <preConditions onFail="MARK_RAN">
+            <rowCount schemaName="alert" tableName="configuration_proxy" expectedRows="0"/>
+        </preConditions>
+        <sql dbms="postgresql" stripComments="true">
+            INSERT INTO alert.configuration_proxy (created_at, last_updated, host, port, username, password)
+            SELECT now(),
+                   null,
+                   'initial-proxy-config',
+                   null,
+                   null,
+                   null
+            FROM GET_GLOBAL_CONFIG_SOURCE_KEY_AND_FIELD_VALUE('component_settings') configFields
+            WHERE configFields.source_key IS NOT NULL LIMIT 1;
+        </sql>
+    </changeSet>
```

Comment:
If there was no proxy configuration previously, do we have any change set to delete this entry if it is all null values? Or do we need this dummy entry for some reason?

Reply:
Maybe each of these should have a precondition to check if a proxy setting exists originally? If proxy fields were shared with other fields in its descriptor then this may be more challenging

Reply:
I believe the GET_GLOBAL_CONFIG_SOURCE_KEY_AND_FIELD_VALUE will return an empty table if the proxy config doesn't exist.  In that case no records will be inserted into the table.  Only if an old proxy configuration exists will this query insert a record into the table.